checkout-ui-custom/src/utils/submitDeliveryForm.js

```diff
@@ -37,6 +37,15 @@ const submitDeliveryForm = async (event) => {
 
   fullAddress = { ...address, ...dbAddress };
 
+  let shouldPersistMasterData = false;
+  // Check for null geoCoordinate and set default value if necessary
+  if (fullAddress.geoCoordinate === null) {
+    shouldPersistMasterData = true;
```

Comment:
never the case

Reply:
@shaheen-karodia as in , it will never go inside that if statement?

Reply:
as in we should not persist the address to master data, only the orderform applixation is required